test/cpp/jit/test_misc.cpp

```diff
@@ -1456,79 +1456,6 @@ TEST(TestSymInt, AddSymbolicInt) {
   ASSERT_TRUE((a + b).expect_int() == 8);
 }
 
-#ifndef C10_MOBILE
-class TestSymNodeImpl : public c10::SymNodeImpl {
- public:
-  explicit TestSymNodeImpl(int64_t i) : i_(i) {}
-
-  bool is_int() override {
-    return true;
-  };
-
-  bool is_float() override {
-    return false;
-  };
-
-  bool bool_() override {
-    return static_cast<bool>(i_);
-  };
-
-#define OPDEF3(NAME, OP, RET)                                         \
-  RET NAME(const c10::SymNode& other) override {                      \
-    return make_intrusive<TestSymNodeImpl>(                           \
-        this->i_ OP dynamic_cast<TestSymNodeImpl*>(other.get())->i_); \
-  }
-
-#define OPDEF2(NAME, OP) OPDEF3(NAME, OP, c10::SymNode)
-  OPDEF2(add, +)
-  OPDEF2(sub, -)
-  OPDEF2(mul, *)
-  OPDEF2(floordiv, /)
-  OPDEF2(mod, %)
-
-  OPDEF2(eq, ==)
-  OPDEF2(ne, !=)
-  OPDEF2(lt, <)
-  OPDEF2(le, <=)
-  OPDEF2(gt, >)
-  OPDEF2(ge, >=)
-#undef OPDEF2
-#undef OPDEF3
-
-  int64_t i_;
-};
-
-TEST(TestSymInt, TestSymIntToSymNodeDispatch) {
-  auto get = [](c10::SymInt si) {
-    auto node = si.toSymNodeImpl();
-    return dynamic_cast<TestSymNodeImpl*>(node.get())->i_;
-  };
-
-  std::vector<int64_t> inputs{0, 1, -1, 4, -4, 777, -777};
-  for (auto i : inputs) {
-    for (auto j : inputs) {
-      auto a = c10::SymInt(
-          static_cast<SymNode>(c10::make_intrusive<TestSymNodeImpl>(i)));
-      auto b = c10::SymInt(
-          static_cast<SymNode>(c10::make_intrusive<TestSymNodeImpl>(j)));
-      ASSERT_EQ(get(a + b), i + j);
-      ASSERT_EQ(get(a - b), i - j);
-      ASSERT_EQ(get(a * b), i * j);
-      if (j != 0) {
-        ASSERT_EQ(get(a / b), i / j);
-        ASSERT_EQ(get(a % b), i % j);
-      }
-      ASSERT_EQ(a == b, i == j);
-      ASSERT_EQ(a != b, i != j);
-      ASSERT_EQ(a < b, i < j);
-      ASSERT_EQ(a <= b, i <= j);
-      ASSERT_EQ(a > b, i > j);
-      ASSERT_EQ(a >= b, i >= j);
-    }
-  }
-}
-#endif
```

Comment:
It would have been pretty annoying to make this test work (since I need to have SymBool tagging now), and it didn't seem like it was buying us much now.